speechtotextv1/websocket_listener.go

```diff
@@ -0,0 +1,109 @@
+package speechtotextv1
+
+import (
+	"encoding/json"
+	"github.com/gorilla/websocket"
+	core "github.com/watson-developer-cloud/go-sdk/core"
+	"io"
+	"time"
+)
+
+type WebsocketListener struct {
```

Comment:
`RecognizeListener` would be better here and add a method `NewWebsocketListener `